code/modules/mining/aux_base_camera.dm

```diff
@@ -24,15 +24,6 @@
 	name = "internal RCD"
 	max_matter = 600 //Bigger container and faster speeds due to being specialized and stationary.
 	no_ammo_message = "<span class='warning'>Internal matter exhausted. Please add additional materials.</span>"
-	walldelay = 10
```

Comment:
Set the general delay to 10 or 25 please, it is intended to be for fast building.